client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -157,8 +152,9 @@ class DesignTypeWithStoreStep extends Component {
 						isVisible={ this.state.showStore }
 					/>
 				</div>
+
 				<div className={ designTypeListClassName }>
-					{ this.getChoices().map( this.renderChoice ) }
+					<TileGrid>{ this.getChoices().map( this.renderChoice ) }</TileGrid>
```

Comment:
Choices appear to be a static array mapped over here. If that's true, simply including the `Tile`s here with their props would be an improvement.

Are you planning to follow up with that?

Reply:
Yeah, that's a good thing to follow-up with in a separate PR (it occurs in other locations that this PR doesn't touch, too). That's one of the things I'm planning to follow-up on after this one gets merged.